src/anupSJun20/AllDifferIndex.java

```diff
@@ -0,0 +1,33 @@
+package anupSJun20;
+/* Program 5: Find all differ index from given arrays.
+               i/p:- arr1: {"Maulik", "Harsh", "Krishna", "Technocredits"}
+                       arr2: {"Maulik","Harsh","Technocredits","Krishna"}		
+               o/p:- Values are not matching at index -> 2, [arr1 value is Krishna, arr2 value is Technocredits] 
+                        Values are not matching at index -> 3, [arr1 value is Technocredits, arr2 value is Krishna] */
+
+public class AllDifferIndex {
+	static void findAllDifferIndex(String[] arr1,String[] arr2) {
+		boolean flag = true;
+		if(arr1.length == arr2.length) {
+			for(int index = 0;index<arr1.length;index++) {
+				if(!arr1[index].equalsIgnoreCase(arr2[index])) {
+						flag = false;
+						System.out.println("Values are not matching at index -> " +index+ " arr1 value is "+ arr1[index] +","+ " arr2 value is "+ arr2[index]);
+					}
+				}
+			if (flag == true) {
```

Comment:
No need to use this variable here, because our motto is to check findAllDifferIndex. Please apply this change in every method.